eventkit_cloud/ui/static/ui/app/components/DataPackPage/DataPackGridItem.js

```diff
@@ -7,8 +7,8 @@ import MenuItem from 'material-ui/MenuItem';
 import moment from 'moment';
 import { List, ListItem } from 'material-ui/List';
 import NavigationMoreVert from 'material-ui/svg-icons/navigation/more-vert';
-import SocialGroup from 'material-ui/svg-icons/social/group';
-import SocialPerson from 'material-ui/svg-icons/social/person';
+import Lock from 'material-ui/svg-icons/action/lock';
```

Comment:
I think in the new DataPack sharing design we will be using Lock from `'material-ui/svg-icons/action/lock-outline'` (for our new private icon). If you want you can make that change in this branch, or just leave it as is and the permissions_management branch will include the updated icons anyway.